packages/block-library/src/query-title/index.php

```diff
@@ -5,6 +5,28 @@
  * @package WordPress
  */
 
+/**
+ * Filters the prefix from the `core/query-title` block archive variation.
+ *
+ * @param array $title Archive title to be displayed.
+ *
+ * @return string Returns the query title based on the queried object but without prefix.
+ */
+function block_query_title_filter_archive_title( $title ) {
+	if ( is_category() ) {
+		return single_cat_title( '', false );
+	} elseif ( is_tag() ) {
+		return single_tag_title( '', false );
+	} elseif ( is_author() ) {
+		return get_the_author();
+	} elseif ( is_post_type_archive() ) {
+		return post_type_archive_title( '', false );
+	} elseif ( is_tax() ) {
+		return single_term_title( '', false );
+	}
```

Comment:
These don't need to be `elseif` 😉 
If we're returning in the lines before, then processing stops at the `return` and does not proceed further down. These can be converted to simple `if` statements 👍 

```suggestion
	}
	if ( is_tag() ) {
		return single_tag_title( '', false );
	}
	if ( is_author() ) {
		return get_the_author();
	}
	if ( is_post_type_archive() ) {
		return post_type_archive_title( '', false );
	}
	if ( is_tax() ) {
		return single_term_title( '', false );
	}
```

Reply:
Since we want to remove **all** archive title prefixes, I removed the function with the if statements.